### PR TITLE
Add ancient flame module to Preservation Evoker guide

### PR DESCRIPTION
**Pull request by trevorm4**
### Description

Adds module/guide section for ancient flame when spec'd into eb talents since its very useful for dps and generating eb procs. Also fixes the gcd configurations for empowered abilities (they have a .5 gcd) and fixes gcd error with living flame.

### Testing

- Test report URL: `report/f2MTV1hH3zZjWk47/28-Mythic+Rashok,+the+Elder+-+Kill+(5:47)/Widdeldwagon/standard/overview`
- Screenshot(s):

image
image
image

### CODE REVIEW

File: src/analysis/retail/evoker/preservation/modules/features/Abilities.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ class Abilities extends CoreAbilities {
         category: SPELL_CATEGORY.ROTATIONAL_AOE,
         cooldown: 30,
         gcd: {
-          base: 1500,
+          base: 500,
```
Comment (vohrr) [inline src/analysis/retail/evoker/preservation/modules/features/Abilities.tsx:33]: Does Fire Breath need to be added & included here

File: src/analysis/retail/evoker/preservation/modules/talents/AncientFlame.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+  get applyTime() {
+    return this.lastApply?.timestamp || this.owner.fight.start_time;
+  }
+
+  onRefresh(event: RefreshBuffEvent) {
+    let tooltip = null;
+    let value = QualitativePerformance.Good;
+    // 1 stack after cast = they were at 2 so not a waste
+    if (this.selectedCombatant.getBuffStacks(SPELLS.ESSENCE_BURST_BUFF.id) === 1) {
```
Comment (vohrr) [inline src/analysis/retail/evoker/preservation/modules/talents/AncientFlame.tsx]: Probably need to track last proccing cast, this misses the refresh event triggered by verdant embrace w/1 EB stack which would be a bad cast. 

1 Stack Essence Burst & Ancient Flame Active  -> VE Cast -> Bad Refresh

Or at the very least the tooltip would be inaccurate for this scenario

File: src/analysis/retail/evoker/preservation/modules/talents/AncientFlame.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    };
+    const styleObjInner = {
+      fontSize: 15,
+    };
+    const explanation = (
+      <p>
+        When playing an <SpellLink spell={SPELLS.EMERALD_BLOSSOM} /> focused build, it is extremely
+        important to weave in casts of <SpellLink spell={SPELLS.LIVING_FLAME_CAST} /> between casts
+        of <SpellLink spell={SPELLS.EMERALD_BLOSSOM} /> in order to generate{' '}
+        <SpellLink spell={TALENTS_EVOKER.ESSENCE_BURST_PRESERVATION_TALENT} />. Not utilizing this
```
Comment (vohrr) [inline src/analysis/retail/evoker/preservation/modules/talents/AncientFlame.tsx:115]: "Not utilizing this talent will reduce your HPS, DPS, and Mana efficiency"

File: src/analysis/retail/evoker/preservation/modules/talents/AncientFlame.tsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+        talent will lead to both an HPS and DPS loss, while also making the build more mana-starved
+        than it should be. It is okay to cast multiple <SpellLink spell={SPELLS.EMERALD_BLOSSOM} />s
```
Comment (vohrr) [inline src/analysis/retail/evoker/preservation/modules/talents/AncientFlame.tsx]: Specify when it is ok (cast at max stacks/high dmg etc.)

**Review by vohrr — CHANGES_REQUESTED** [review]

File: src/analysis/retail/evoker/preservation/modules/features/Abilities.tsx (hunk 1 above)
Comment (trevorm4) [reply src/analysis/retail/evoker/preservation/modules/features/Abilities.tsx:33]: Fire Breath is in the shared abilities module

**Comment by trevorm4** [comment]
Addressed comments

**Review by vohrr — APPROVED** [review]

**Pull request merged**